packages/minter-contracts/ligo/src/fixed_price_sale/README.md

```diff
@@ -0,0 +1,265 @@
+# Fixed Price Sale
+
+The contracts titled `fixed_price_sale_market.mligo` and
+`fixed_price_sale_market_tez.mligo` both contain implementations of a sale of a
+non-fungible token (NFT) at a fixed price. The former contract allows a seller
+to sell an NFT for some fixed price of FA2 tokens. The latter instead allows a
+seller to sell an NFT for some fixed price of tezzies. Although, the general
+design and implementation of this contract is fairly straightforward, someone
+unfamiliar with the Tezos blockchain's inner workings needs to be aware of
+certain key items. These details will discussed in the following section
+[Entrypoints](#entrypoints-section). For now, we shall discuss the storage
+contents of each contract.
+
+## <a name="storage-section"></a> Storage
+
+The next two subsections will list and explain the storage contents of each
+contract. For the sake of concision, we hereby label the contract
+`fixed_price_sale_market.mligo` as `SALE_FA2` and the other
+(`fixed_price_sale_market_tez.mligo`) as `SALE_TEZ`.
+
+### <a name="sale-fa2-section"></a>`SALE_FA2` Storage
+
+The contract's storage is:
+
+``` ocaml
+type storage = {
+    admin: pauseable_admin_storage;
+    sales: (sale_param, nat) big_map;
+}
+```
+
+The first field of this record is `admin`. Its type is:
+
+``` ocaml
+type pauseable_admin_storage_record = {
+  admin : address;
+  pending_admin : address option;
+  paused : bool;
+}
+
+type pauseable_admin_storage = pauseable_admin_storage_record option
+```
+
+A thorough explication of this type's semantics will delayed for later in this
+document. For now, simply note that this type endows this contract with
+administrative operations. Namely, these are the abilities to pause operations
+and prevent entrypoint invocation by unauthorized callers (if desired).
+
+The second field of this record `sales`is simply a "record" of currently active
+sales that the fixed price sale contract is hosting. To be clear, when we use
+the phrase "currently active sale", we mean to say that a seller has initiated
+some sale and there has of yet been no action taken that would prevent someone
+from buying the sale's offered product. The value of each entry in the big_map
+is the number of FA2 tokens that the seller specifies as the NFT's value. The
```

Comment:
Not quite clear what is the values in that big_map

Reply:
I thought I said this in the sentence "The value of each entry in the big_map is the number of FA2 tokens that the seller specifies as the NFT's value." Is there something unclear about this sentence?